src/model/guild/guild_id.rs

```diff
@@ -909,7 +907,7 @@ impl<'a> From<&'a Guild> for GuildId {
 
 /// A helper class returned by [`GuildId::members_iter`]
 #[derive(Clone, Debug)]
-#[cfg(all(feature = "model", feature = "cache"))]
+#[cfg(all(feature = "model"))]
```

Comment:
As we're checking for just one feature, the `all` predicate can be dropped.
```suggestion
#[cfg(feature = "model")]
```